managed/src/main/resources/v1.routes

```diff
@@ -17,6 +17,7 @@ PUT     /customers/:cUUID/api_token                                            c
 GET     /logout                                                                com.yugabyte.yw.controllers.SessionController.logout()
 + nocsrf
 POST    /register                                                              com.yugabyte.yw.controllers.SessionController.register()
+GET     /validate                                                              com.yugabyte.yw.controllers.SessionController.validate()
```

Comment:
Can we update the naming to be more specific? Should be something like `password_policy` not `validate`, which implies a POST request.

Reply:
@yb-andrew : I have resolved the issue, changed the naming part and tested it locally. Please re-review, thanks !